arrow-fx/src/main/kotlin/arrow/fx/MVar.kt

```diff
@@ -168,7 +168,7 @@ interface MVar<F, A> {
   fun tryTake(): Kind<F, Option<A>>
 
   /**
-   * Tries reading the current value, or blocks until there is a value available.
+   * Reads the current value without touching it, assuming there is one, or otherwise it blocks until there is a value available.
```

Comment:
```suggestion
   * Reads the current value without emptying the MVar, assuming there is one, or otherwise it blocks until there is a value available.
```